docs/querying/basics.md

```diff
@@ -218,30 +251,31 @@ For example, the following expression returns the value of
 Note that the `@` modifier always needs to follow the selector
 immediately, i.e. the following would be correct:
 
-    sum(http_requests_total{method="GET"} @ 1609746000) // GOOD.
+    sum( http_requests_total{method="GET"} @ 1609746000 ) // GOOD.
 
 While the following would be *incorrect*:
 
-    sum(http_requests_total{method="GET"}) @ 1609746000 // INVALID.
+    sum( http_requests_total{method="GET"} ) @ 1609746000 // INVALID.
```

Comment:
Please do not change the formatting here, I think it reads better without the spaces.

Reply:
Same as above, agreed.